### ISSUE TITLE
The wp_frm_payments' doesn't exist

### ISSUE THREAD

**Issue by arshidkv12** [Strategy11/formidable-forms#1292]
`[wp_frm_payments' doesn't exist]
SELECT * FROM wp_frm_payments WHERE item_id = '4' ORDER BY created_at DESC`

Version: 6.5 (Lite version)

**Comment by Crabcyborg** [comment]
Thank you for reporting this issue @arshidkv12

I've made a small update to address this issue for next release https://github.com/Strategy11/formidable-forms/pull/1293

**Comment by jas8522** [comment]
Every 7 days wp-cron reports a similar issue:

```
WordPress database error Table 'DB.wp_frm_payments' doesn't exist for query SELECT amount, status, paysys, created_at FROM wp_frm_payments made by do_action_ref_array('formidable_send_usage'), WP_Hook->do_action, WP_Hook->apply_filters, FrmUsageController::send_snapshot, FrmUsage->send_snapshot, FrmUsage->snapshot, FrmUsage->payments
WordPress database error Table 'DB.wp_frm_subscriptions' doesn't exist for query SELECT amount, status, paysys, created_at FROM wp_frm_subscriptions made by do_action_ref_array('formidable_send_usage'), WP_Hook->do_action, WP_Hook->apply_filters, FrmUsageController::send_snapshot, FrmUsage->send_snapshot, FrmUsage->snapshot, FrmUsage->payments
```

**Comment by Crabcyborg** [comment]
Hi @jas8522.

Thank you for reporting.

This issue should actually be fixed in version 6.16.3.

I recommend updating when you have the chance.

**Comment by jas8522** [comment]
Thank you!
